Documentation/Settings.cfg

```diff
@@ -0,0 +1,96 @@
+# #####
+#
+# Settings.cfg - A TYPO3 Documentation Project's Configuration File
+# Information about Settings.cfg:
+#   https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/GeneralConventions/DirectoryFilenames.html#settings-cfg
+#
+# About Syntax:
+#   See https://docs.python.org/2/library/configparser.html
+#
+# Attention:
+#   Only " ;" can start an inline comment.
+#   This is: blank PLUS semicolon!
+#
+# #####
+
+[general]
+
+# .................................................................................
```

Comment:
The following settings might by modified. I'm not sure if this information is correct.

Reply:
changed with commit be28db3490460d84eb6aefc6d96a12936f2a6c23